apps/core0/subsys/containers/filesystem.go

```diff
@@ -163,7 +185,7 @@ func (c *container) sandbox() error {
 				return err
 			}
 			//assume a flist
-			if err := c.mountFList(src, target); err != nil {
+			if err := c.mountFList(src, target, cfg); err != nil {
```

Comment:
no, remove this from here please. Overrides happens on the `/` of the container only